packages/go_router_builder/lib/src/go_router_generator.dart

```diff
@@ -18,7 +18,7 @@ class GoRouterGenerator extends GeneratorForAnnotation<void> {
 
   @override
   TypeChecker get typeChecker => const TypeChecker.fromUrl(
-        'package:go_router/src/route_data.dart#TypedGoRoute',
+        'package:go_router/src/typed_routing.dart#TypedGoRoute',
```

Comment:
typedgoroute is exposed as a public class, this should be 'package:go_router/go_router.dart#TypedGoRoute`

Reply:
Hmm, this doesn't seem to work